packages/rum/src/domain/record/observers.ts

```diff
@@ -151,23 +152,25 @@ export function initMutationObserver(
   return startMutationObserver(cb, configuration, shadowRootsController, document)
 }
 
-function initMoveObserver(cb: MousemoveCallBack): ListenerHandler {
+export function initMoveObserver(cb: MousemoveCallBack): ListenerHandler {
   const { throttled: updatePosition } = throttle(
     monitor((event: MouseEvent | TouchEvent) => {
       const target = getEventTarget(event)
       if (hasSerializedNode(target)) {
-        const { clientX, clientY } = isTouchEvent(event) ? event.changedTouches[0] : event
+        const coordinates = tryComputeCoordinates(event)
+        if (!coordinates) {
+          addTelemetryDebug('mouse/touch event without x/y', {
+            isTrusted: event.isTrusted,
+          })
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ ‏you could also move the telemetry debug in `tryComputeCoordinates`